assets/sass/_content.scss

```diff
@@ -374,12 +375,12 @@ body.custom-background {
           float: right;
           text-align: right;
 
-          @media screen and (max-width: $width-phone) {
+          @include max-width($width-phone) {
             float: none;
           }
         }
 
-        @media screen and (max-width: $width-phone) {
+        @include max-width($width-phone) {
```

Comment:
Expected item on line 383 to appear before line 371. Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets